decidim-core/app/views/layouts/decidim/header/_redesigned_menu_breadcrumb_main_dropdown.html.erb

```diff
@@ -6,12 +6,11 @@
     <%= breadcrumb_root_menu.render %>
   </div>
   <div class="menu-bar__main-dropdown__bottom">
-  <%# REDESIGN_PENDING - This element should use a cell and define a criteria to select a single featured element %>
     <div class="menu-bar__main-dropdown__bottom-left">
-      <%= render partial: "layouts/decidim/header/redesigned_menu_breadcrumb_main_dropdown_bottom_left" %>
+      <%= cell("decidim/highlighted_participatory_process", menu_highlighted_participatory_process) %>
     </div>
     <div class="menu-bar__main-dropdown__bottom-right">
-      <%= cell "decidim/content_blocks/menu_breadcrumb_last_activity", nil %>
+      <%= cell "decidim/content_blocks/menu_breadcrumb_last_activity", defined?(current_participatory_space) ? current_participatory_space : current_organization %>
```

Comment:
If we want to show the last activity of the whole page in the menu we should change this:

```suggestion
      <%= cell "decidim/content_blocks/menu_breadcrumb_last_activity", current_organization %>
```